Formula/zsh.rb

```diff
@@ -29,11 +29,18 @@ class Zsh < Formula
     sha256 "130c8b5367cd7f94f6ff0de8a6d02c723671263a5e49e361dd502b3dbc737073" => :yosemite
   end
 
+  devel do
+    url "https://www.zsh.org/pub/development/zsh-5.3.1-test-2.tar.gz"
+    version "5.3.1-test-2"
```

Comment:
Wait a second. `5.3.1-test-2` is a release candidate for `5.4`. `5.3.1` is the current stable. Isn't this expected

```rb
[5] brew(main)> Version.new("5.3.1-test-2") > Version.new("5.3.1") && Version.new("5.3.1-test-2") < Version.new("5.4")
=> true
```